src/extension/background-script/connectors/lndhub.ts

```diff
@@ -352,6 +352,10 @@ export default class LndHub implements Connector {
 
       if (axios.isAxiosError(e)) {
         const errResponse = e.response as AxiosResponse;
+        if (errResponse?.status === 404) {
+          const method = path.replace("/", "");
+          throw new Error(`${method} not supported by the connected account.`);
```

Comment:
Will the user be informed by this as well?

Reply:
Yes, the user will see this in the new Toast